<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeSuit</class>
 <widget class="QDialog" name="ChangeSuit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="spadePic">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Spades&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="diamondPic">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Diamonds&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="clubPic">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Clubs&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="heartPic">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>90</width>
     <height>130</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Hearts&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="spades">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="hearts">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="clubs">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="diamonds">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>318</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>spadePic</zorder>
  <zorder>diamondPic</zorder>
  <zorder>clubPic</zorder>
  <zorder>heartPic</zorder>
  <zorder>spades</zorder>
  <zorder>hearts</zorder>
  <zorder>clubs</zorder>
  <zorder>diamonds</zorder>
  <zorder>choose</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
